app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/GroupsFragment.kt

```diff
@@ -1,15 +1,15 @@
 package edu.uc.hickmadc.customnotifications.ui.main
 
 import android.os.Bundle
-import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
 import edu.uc.hickmadc.customnotifications.adapters.GroupsAdapter
 import edu.uc.hickmadc.customnotifications.databinding.FragmentGroupsBinding
 import edu.uc.hickmadc.customnotifications.dto.Group
-import java.util.ArrayList
+import java.util.*
```

Comment:
Not sure why it decided to import all of java.util.

Reply:
For documentation, it's best to avoid using the * in import statements.